neqo-http3/src/lib.rs

```diff
@@ -341,6 +332,12 @@ pub trait HttpRecvStream: RecvStream {
     /// # Errors
     /// An error may happen while reading a stream, e.g. early close, protocol error, etc.
     fn read_data(&mut self, conn: &mut Connection, buf: &mut [u8]) -> Res<(usize, bool)>;
+
+    fn priority(&self) -> Priority;
+
+    fn priority_update(&mut self, priority: Priority);
+    fn priority_update_outstanding(&self) -> bool;
+    fn priority_update_sent(&mut self);
```

Comment:
these 4 functions manipulate priorities. 
What about having a PriorityHandler that implements last 3 functions? and HttpRecvStream has only:

`priority(&mut self) -> &'a mut PriorityHandler`